browser-extension/src/browserAction/menu.js

```diff
@@ -24,7 +25,7 @@ function get_current_tab_video_id() {
  */
 function openTournesolHome() {
   chrome.tabs.create({
-    url: 'https://tournesol.app?utm_source=extension&utm_medium=menu',
+    url: `${frontendUrl}?utm_source=extension&utm_medium=menu`,
```

Comment:
**minor**

Here we replace `https://tournesol.app` by `https://tournesol.app/`.

I still need to check but I think our web analytics software will correctly consider both URL as the same home page. Unfortunately this is not the case of other pages. For instance `https://tournesol.app/recommendations` and `https://tournesol.app/recommendations/` will be considered as two different pages.

Even if this is not a problem for the Tournesol home page, I wonder if using a trailing slash here will not invite future contributors to write front end URL in `prepareExtention.js` with a trailing slash, which will create duplicates in our web analytics dashboard.

Maybe a comment in the `prepareExtention.js` would be enough? Maybe a comment and also removing the trailing slash on the home page URL is better?

```javascript

// please DO NOT add a trailing slash to front end URL, this prevents
// creating duplicates in our web analytics tool
const config = getForEnv(
  {
    production: {
      frontendUrl: 'https://tournesol.app',
      frontendHostEquals: 'tournesol.app',
      apiUrl: 'https://api.tournesol.app/',
    },
    'dev-env': {
      frontendUrl: 'http://localhost:3000',
      frontendHostEquals: 'localhost:3000',
      apiUrl: 'http://localhost:8000/',
    },
  },
  env
);
```

Reply:
Done in 928ad8d82deaba547b188184128c6a584b46298e.